src/scrabble.js

```diff
@@ -1,5 +1,82 @@
-function scrabble() {
-
+function scrabble(str) {
+  let counter = 0 // must be local to restart and redefine back to 0 when finished otherwise global scope will add it up
+  if (!str || str.length === 0) {
+    return 0
+  }
+  for (let i = 0; i < str.length; i++) {
+    if (
+      str[i] === 'a' ||
```

Comment:
Great job on getting all these comparisons in and looking nice! I took a different approach by creating arrays for each score value and each array including the letters associated with that score. But this works brilliantly!